projects/plugins/jetpack/tests/e2e/lib/flows/log-in.js

```diff
@@ -25,7 +25,7 @@ export async function loginToWpSite( mockPlanData ) {
 	}
 }
 
-export async function loginToWpCom( wpComUser, mockPlanData ) {
+export async function loginToWpCom( mockPlanData ) {
```

Comment:
This approach kinda removes the flexibility of this function. Now it would not be possible to call it within the test. I'm fine with that, since it is not something we do now, and if needed, we could update the fuction.